README.md

```diff
@@ -1,5 +1,6 @@
 To start hacking on Hydra, run:
 
+    $ bootstrap
```

Comment:
It would be better to run `$configurePhase`, because this is part of `preConfigure` in release.nix